src/librustdoc/lib.rs

```diff
@@ -193,6 +199,24 @@ pub fn usage(argv0: &str) {
                             &opts()));
 }
 
+fn is_unstable_flag_enabled(nightly_build: bool, has_z_unstable_options: bool,
+                                  flag_name: &str) {
+    // check if unstable for --extend-css option
+    match if !nightly_build {
```

Comment:
nit: 
- maybe this could just be a `let` instead of `match` and early return in the `else` case?
- could this function be named `check_*`? With `is_*` I'd expect a bool return value.